modules/security-context-constraints-about.adoc

```diff
@@ -5,58 +5,81 @@
-be more permissive.
-////
-
-The cluster contains nine default SCCs:
-
-* `anyuid`
-* `hostaccess`
-* `hostmount-anyuid`
-* `hostnetwork`
-+
-[WARNING]
-====
-If additional workloads are run on control plane hosts (also known as the master hosts), use caution when providing access to `hostnetwork`. A workload that runs `hostnetwork` on a control plane host is effectively root on the cluster and must be trusted accordingly.
-====
-* `node-exporter`
-* `nonroot`
-* `privileged`
-* `restricted`
-* `pipelines-scc`
+Similar to the way that RBAC resources control user access, administrators can use _security context constraints_ (SCCs) to control permissions for pods. These permissions include actions that a pod can perform and what resources it can access. You can use SCCs to define a set of conditions that a pod must run with to be accepted into the system.
+
+Security context constraints allow an administrator to control:
 
+* Whether a pod can run privileged containers
+* The capabilities that a container can request
+* The use of host directories as volumes
+* The SELinux context of the container
+* The container user ID
+* The use of host namespaces and networking
+* The allocation of an `FSGroup` that owns the pod volumes
+* The configuration of allowable supplemental groups
+* Whether a container requires the use of a read only root file system
+* The usage of volume types
+* The configuration of allowable `seccomp` profiles
+
+[id="default-sccs_{context}"]
+== Default security context constraints
+
+The cluster contains several default security context constraints (SCCs) as described in the table below. By default, all pods and containers use the `restricted` SCC.
 
 [IMPORTANT]
 ====
 Do not modify the default SCCs. Customizing the default SCCs can lead to issues when some of the platform pods deploy or {product-title} is upgraded. During upgrades between some versions of {product-title}, the values of the default SCCs are reset to the default values, which discards all customizations to those SCCs.
 ifdef::openshift-origin,openshift-enterprise,openshift-webscale[]
-Instead, create new SCCs.
+Instead, create new SCCs as needed.
 endif::[]
 ====
 
+.Default security context constraints
+[cols="1,3a",options="header"]
+|===
+|Security context constraint |Description
+
+|`anyuid`
+| Provides all features of the `restricted` SCC, but allows users to run with any UID and any GID.
+
+|`hostaccess`
+|Allows access to all host namespaces but still requires pods to be run with a UID and SELinux context that are allocated to the namespace. WARNING: this SCC allows host access to namespaces, file systems, and PIDS.  It should only be used by trusted pods.  Grant with caution.
+
+|`hostmount-anyuid`
+|Provides all the features of the `restricted` SCC, but allows host mounts and running as any UID and any GID on the system. This is primarily used by the persistent volume recycler.
+
+[WARNING]
```

Comment:
Can 'NOTE' be used here? Is there rule for the usage such as WARNING, NOTE and IMPORTANT?

Reply:
NOTE is usually for basic information purposes, so I do think that WARNING is more appropriate here since there could be a security risk if they weren't aware of the permissions these SCCs have.